google/resource_bigquery_dataset_test.go

```diff
@@ -112,3 +159,97 @@ resource "google_bigquery_dataset" "test" {
   }
 }`, datasetID)
 }
+
+func testAccBigQueryDatasetWithOneAccess(datasetID string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "access_test" {
+  dataset_id                  = "%s"
+
+  access {
+    role          = "OWNER"
+    user_by_email = "Joe@example.com"
+  }
+
+  labels {
+    env                         = "foo"
+    default_table_expiration_ms = 3600000
+  }
+}`, datasetID)
+}
+
+func testAccBigQueryDatasetWithTwoAccess(datasetID string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "access_test" {
+  dataset_id                  = "%s"
+
+  access {
+    role          = "OWNER"
+    user_by_email = "Joe@example.com"
+  }
+  access {
+    role	      = "READER"
+    domain	      = "example.com"
+  }
+
+  labels {
+    env                         = "foo"
+    default_table_expiration_ms = 3600000
+  }
+}`, datasetID)
+}
+
+func getBigQueryTableWithView(datasetID, tableID string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "other_dataset" {
+  dataset_id = "%s"
+}
+
+resource "google_bigquery_table" "table_with_view" {
+  table_id   = "%s"
+  dataset_id = "${google_bigquery_dataset.other_dataset.dataset_id}"
+
+  time_partitioning {
+    type = "DAY"
+  }
+
+  view {
+        query = "SELECT state FROM [lookerdata:cdc.project_tycho_reports]"
```

Comment:
nit: # of spaces consistency with the rest of the resource

Reply:
Done.